yarn-project/end-to-end/src/e2e_offchain_note_delivery.ts

```diff
@@ -0,0 +1,51 @@
+import { Fr, type Wallet } from '@aztec/aztec.js';
+import { MAX_NOTE_HASHES_PER_TX } from '@aztec/constants';
+import { TestContract } from '@aztec/noir-contracts.js/Test';
+
+import { setup } from './fixtures/utils.js';
+
+describe('e2e_offchain_note_delivery', () => {
```

Comment:
This is a renaming of the old e2e_non_contract_account, which was actually unrelated to non contract accounts and just contained this. I think the test is valuable, given that it'll be the only example of it if we remove the token blacklist, so I kept it.

Reply:
Yes, imminently.